airbyte-integrations/connectors/source-github/acceptance-test-config.yml

```diff
@@ -0,0 +1,44 @@
+connector_image: airbyte/source-github:dev
+tests:
+  spec:
+    - spec_path: "source_github/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      validate_output_from_all_streams: yes
+    # Below streams have too much records and because of it we are getting timeouts and hitting rate limits.
```

Comment:
Is it still need?

Reply:
I believe yes. This comment in pair with `too_large_streams_configured_catalog.json` file will tell developers why tests for those streams are turned off.